### PR TITLE
mcobbett copyright build pipelines

### PR DESCRIPTION
**Pull request by techcobweb**
- Working on the github copyright app pipelines still..
- Another attempt
- Stopping here for now
- Adding comments to explain progress so far
- Remove task that was needed for testing purposes
- revert make task to what it was in main
- Make with params

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: pipelines/pipelines/githubapp-copyright/build-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      value: $(params.prUrl)
+    - name: action
+      value: $(params.action)
+# 
+# 
+# 
+  - name: clone-githubapp-copyright
+    taskRef: 
+      name: git-clone
+    params:
```
Comment (jadecarino) [inline pipelines/pipelines/githubapp-copyright/build-pr.yaml:55]: Can you add this in here:
```
runAfter:
      - git-verify
```

**Review by jadecarino — CHANGES_REQUESTED** [review]

File: pipelines/pipelines/githubapp-copyright/build-pr.yaml (hunk 1 above)
Comment (techcobweb) [reply pipelines/pipelines/githubapp-copyright/build-pr.yaml:57]: Well spotted. Fixed.

**Review by jadecarino — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**